commit 3f1dd3b94ab2b56abb43a51b7b78a9cd162325d0
Author: Aaron Seigo <aseigo@kde.org>
Date:   Fri May 6 22:51:50 2011 +0200

    get rid of duplicate connect
    
    patch contributed by Luc Menut
    BUG:262614
    REVIEW:101284

diff --git a/plasma/generic/containmentactions/applauncher/launch.cpp b/plasma/generic/containmentactions/applauncher/launch.cpp
index cbc7175..bef53e5 100644
--- a/plasma/generic/containmentactions/applauncher/launch.cpp
+++ b/plasma/generic/containmentactions/applauncher/launch.cpp
@@ -36,7 +36,6 @@ AppLauncher::AppLauncher(QObject *parent, const QVariantList &args)
 {
     m_menu = new KMenu();
     connect(m_menu, SIGNAL(triggered(QAction*)), this, SLOT(switchTo(QAction*)));
-    connect(m_menu, SIGNAL(triggered(QAction*)), this, SLOT(switchTo(QAction*)));
 
     m_action->setMenu(m_menu);
 }
